jupyterbook/content/Code Gallery/R_Matlab_Notebooks/2017-08-01-xtractoR.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
It is hard to follow if these diffs are due to directory move or if they really have differences. But that is fine for now. We cannot escape from reviewing the final HTML at some point.